example/.gitignore

```diff
@@ -7,13 +7,15 @@
 build/
 
 .flutter-plugins
+.flutter-plugins-dependencies
 
 android/.project
 android/.settings/org.eclipse.buildship.core.prefs
 android/app/.classpath
 android/app/.project
 android/app/.settings/org.eclipse.buildship.core.prefs
 android/app/bin/
-ios/SquareReaderSDK.framework
+ios/SquareReaderSDK.xcframework
 ios/Runner.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
+ios/flutter
```

Comment:
nit: move this above ios/SquareReaderSDK.xcframework

Reply:
Okay.